SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -34,6 +35,23 @@ USpatialGDKEditorSettings::USpatialGDKEditorSettings(const FObjectInitializer& O
 	SpatialOSSnapshotToLoad = GetSpatialOSSnapshotToLoad();
 }
 
+const FString& USpatialGDKEditorSettings::GetPinnedGDKRuntimeVersion()
+{
+	static const FString PinnedVersion("11-20200205T105018Z-7668e9b");
```

Comment:
Wonder if we should instead stick this inside the `SpatialGDKServices` Module as this is the only module that can be used by all other GDK modules.